src/collections/_documentation/platforms/python/django.md

```diff
@@ -66,3 +66,32 @@ The following keyword arguments can be passed to `DjangoIntegration()`:
 
 The user feedback feature can be used with this integration.  For more information
 see [User Feedback]({% link _documentation/enriching-error-data/user-feedback.md %}?platform=django).
+
+## Reporting other status codes
+
+In some situations it might make sense to report 404s and other errors besides 500 to Sentry. This can be achieved by writing your own Django view for those status codes. For example:
+
+```python
+
+# urls.py
+
+handler404 = 'mysite.views.my_custom_page_not_found_view'
+
+# views.py
+
+from django.http import HttpResponseNotFound
+from sentry_sdk import capture_message
+
+
+def my_custom_page_not_found_view(*args, **kwargs):
+    capture_message("Page not found!", level="error")
+
+    # return any response here
```

Comment:
How would you feel about this:

```
# return any response here, for ex:
return HttpResponseNotFound("Not found")
```
for the last two lines?

Reply:
can do

Reply:
done